great_expectations/datasource/generator/passthrough_generator.py

```diff
@@ -17,10 +17,10 @@ def _get_iterator(self, generator_asset, **kwargs):
         return iter([kwargs])
 
     def get_available_data_asset_names(self):
-        logger.warning(
+        logger.debug(
             "PassthroughGenerator cannot identify data_asset_names, but can accept any object as a valid data_asset."
         )
-        return []
+        return {"names": []}
```

Comment:
This should be changed in *all* generators if we're going to make the change; it's a significant API change.

Reply:
I think I updated all the generators